### PR TITLE
Move inline styles to a file atoms.js

### PR DESCRIPTION
**Pull request by adam-lzk**
Move inline styles to a file atoms.js, but they don't work

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,14 @@
 import { Layout, Menu } from 'antd'
-const { Header, Footer, Sider, Content } = Layout
+const { Header, Footer, Content } = Layout
+import { StyledSider } from './atoms.js'
 import './App.css'
 
 function App() {
   return (
   <Layout>
     <Header className='header'>Khinkalnya</Header>
```
Comment (dmlaziuk) [inline src/App.jsx]: Please remove App.css completely by moving all styles to atoms.js
```suggestion
    <StyledHeader>Khinkalnya</StyledHeader >
```

**Review by dmlaziuk — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx [hunk 2]
```diff
@@ -1,20 +1,13 @@
...
     <Layout>
-      <Sider
-        className='sider'
-        style={{
-          height: 'calc(100% - 64px)',  // viewport height minus header height
-          position: 'sticky',
-          top: '64px',
-        }}
-      >
+      <StyledSider className='sider'>
```
Comment (dmlaziuk) [inline src/App.jsx]: We don't have style `.sider`
```suggestion
      <StyledSider>
```

File: src/App.jsx [hunk 3]
```diff
@@ -26,11 +19,11 @@ function App() {
             { key: "6", label: "Contacts" },
           ]}
         />
-      </Sider>
-      <Content className='content'>
-        <img
-          src="../pictures/pict1.jpg"
-          alt="Khinkali"
+      </StyledSider>
+      <StyledContent className='content'>
```
Comment (dmlaziuk) [inline src/App.jsx]:
```suggestion
      <StyledContent>
```

File: src/App.jsx [hunk 4]
```diff
@@ -39,22 +32,22 @@ function App() {
...
+          src='../pictures/pict2.jpg'
+          alt='Khinkali'
         />
         <p>Everyone is comfortable in Khinkalna: we have a children&apos;s menu and comfortable chairs,
         as well as entertainment for the little guests, our genatsvalikov!</p>
         <p>Dear friends, we invite you to the family of Khinkalnya restaurants in your city — Minsk, Brest,
         Grodno, Mozyr, Soligorsk and Pinsk! Choose your favorite dishes in our menu, and we will serve them
         with we will prepare it with love, pack it well and bring it back. Hot, juicy, fragrant!</p>
-        <Footer className='footer'>
+        <StyledFooter className='footer'>
```
Comment (dmlaziuk) [inline src/App.jsx]:
```suggestion
        <StyledFooter>
```

**Review by dmlaziuk — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 2 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 3 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 4 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

**Review by dmlaziuk — APPROVED** [review]

**Pull request merged**